### ISSUE TITLE
aria-describedby should not map to UIA_DescribedBy on UIA

### ISSUE THREAD

**Issue by benbeaudry** [w3c/core-aam#204]
While they share the same name, the definition of the [UIA_DescribedBy](https://learn.microsoft.com/en-us/windows/win32/winauto/uiauto-automation-element-propids) property actually matches the definition of [aria-details](https://www.w3.org/TR/wai-aria-1.2/#aria-details).

The UIA expectations for the aria-describedby attribute should be only:

Property: FullDescription: <value>
See also: [Name Computation](https://www.w3.org/TR/core-aam-1.2/#mapping_additional_nd)

**Comment by sivakusayan** [comment]
It seems like the [UIAutomation docs](https://learn.microsoft.com/en-us/windows/win32/winauto/uiauto-ariaspecification#w3c-aria-states-and-properties-mapped-to-microsoft-active-accessibility-and-ui-automation) call out that `aria-describedby` maps to UIA `DescribedBy`, although it's possible that table is a little outdated since `aria-channel` is apparently still an entry on there ~16 years after its removal. 

Would it be possible to ask one of the contributors on that article to fix that mapping to map to `FullDescription` instead? Otherwise, I think this could possibly be a point of confusion.

**Comment by benbeaudry** [comment]
Good catch, I've reached out to folks at Microsoft to bring a much-needed update to this doc.

**Comment by spectranaut** [comment]
Test update: https://github.com/web-platform-tests/wpt/pull/42784
